integration/kubernetes/kubeadm/config.yaml

```diff
@@ -21,7 +21,7 @@ scheduler:
 ---
 apiVersion: kubelet.config.k8s.io/v1beta1
 kind: KubeletConfiguration
-cgroupDriver: cgroupfs
```

Comment:
is it feasible to use both cgroup managers in our tests? I'd like to ensure cgroupfs is working well still.

Reply:
It is.  What about testing containerd with cgroupfs (so, no changes on containerd side) and CRI-O using systemd?

Reply:
it'd be something as simple as this follow-up patch (which I'll apply after a first run of the CI passes).
```
fidencio@quino ~/go/src/github.com/kata-containers/tests $ git status
En la rama wip/integration-tests-improvements
Cambios no rastreados para el commit:
  (usa "git add <archivo>..." para actualizar lo que será confirmado)
  (usa "git restore <archivo>..." para descartar los cambios en el directorio de trabajo)
	modificado:     integration/kubernetes/init.sh
	modificado:     integration/kubernetes/kubeadm/config.yaml

sin cambios agregados al commit (usa "git add" y/o "git commit -a")
fidencio@quino ~/go/src/github.com/kata-containers/tests $ git diff
diff --git a/integration/kubernetes/init.sh b/integration/kubernetes/init.sh
index 700c7def..ab2d77ff 100755
--- a/integration/kubernetes/init.sh
+++ b/integration/kubernetes/init.sh
@@ -55,9 +55,11 @@ fi
 case "${cri_runtime}" in
 containerd)
        cri_runtime_socket="/run/containerd/containerd.sock"
+       cgroup_driver="cgroupfs"
        ;;
 crio)
        cri_runtime_socket="/var/run/crio/crio.sock"
+       cgroup_driver="systemd"
        ;;
 *)
        echo "Runtime ${cri_runtime} not supported"
@@ -100,6 +102,7 @@ kubeadm_config_file="$(mktemp --tmpdir kubeadm_config.XXXXXX.yaml)"
 
 sed -e "s|CRI_RUNTIME_SOCKET|${cri_runtime_socket}|" "${kubeadm_config_template}" > "${kubeadm_config_file}"
 sed -i "s|KUBERNETES_VERSION|v${kubernetes_version/-*}|" "${kubeadm_config_file}"
+sed -i "s|CGROUP_DRIVER|${cgroup_driver}|" "${kubeadm_config_file}"
 
 trap 'sudo -E sh -c "rm -r "${kubeadm_config_file}""' EXIT
 
diff --git a/integration/kubernetes/kubeadm/config.yaml b/integration/kubernetes/kubeadm/config.yaml
index ee1f0b92..f6f013ec 100644
--- a/integration/kubernetes/kubeadm/config.yaml
+++ b/integration/kubernetes/kubeadm/config.yaml
@@ -21,7 +21,7 @@ scheduler:
 ---
 apiVersion: kubelet.config.k8s.io/v1beta1
 kind: KubeletConfiguration
-cgroupDriver: systemd
+cgroupDriver: CGROUP_DRIVER
 featureGates:
   PodOverhead: true
 systemReserved:
```